index.html

```diff
@@ -2,35 +2,36 @@
-                <input type="button" class="calculator-form__item number action-bar__button-three" id="three" value="3">
-                <input type="button" class="calculator-form__item operators action-bar__button-sub" id="sub" value="-">
-                <input type="button" class="calculator-form__item number action-bar__button-zero" id="zero" value="0">
-                <input type="button" class="calculator-form__item operators action-bar__button-sum" id="sum" value="+">
-                <input type="button" class="calculator-form__item action-bar__button-result" id="result" value="=">
+        <div class="calculator-container">
+            <div class="calculator">
+                <input type="text" class="calculator__output" disabled>
+                <div class="calculator__action-bar">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__ce" value="CE">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__seven" value="7">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__eight" value="8">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__nine" value="9">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__div" value="&divide">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__four" value="4">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__five" value="5">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__six" value="6">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__mul" value="&times">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__one" value="1">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__two" value="2">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__three" value="3">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__sub" value="-">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__zero" value="0">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__sum" value="+">
+                    <input type="button" class="calculator-action-bar__item action-bar-item__result" value="=">
+                </div>
             </div>
-            <div class="calculator-bottom">
-                <a class="calculator-bottom__href" href="https://github.com/NikitaOstrovsky636">GitHub</a>
+            <div class="calculator__footer">
+                <a class="calculator-footer__github-link" target="blanc" href="https://github.com/NikitaOstrovsky636">GitHub</a>
```

Comment:
I'm pretty sure `blanc` is an incorrect value for `target` attribute. Did you mean `_blank`?